### PR TITLE
get info current device connected

### PR DESCRIPTION
**Pull request by danielgssp**
//Exemple getCurrentDevice

```
GoogleCast.getCurrentDevice().then(state =>
    console.log(state)
);
```

### CODE REVIEW

File: android/src/main/java/com/reactnative/googlecast/GoogleCastModule.java [hunk 1]
```diff
@@ -258,6 +258,21 @@ private void setupCastListener() {
     mSessionManagerListener = new GoogleCastSessionManagerListener(this);
   }
 
+  @ReactMethod
+  public void getCurrentDevice(final Promise promise) {
+    getReactApplicationContext().runOnUiQueueThread(new Runnable() {
+      @Override
+      public void run() {
+        WritableMap map = Arguments.createMap();
+        map.putString("id:", mCastSession.getCastDevice().getDeviceId());
```
Comment (petrbela) [inline android/src/main/java/com/reactnative/googlecast/GoogleCastModule.java]: the key should be without the `:`

**Comment by petrbela** [comment]
Cool! Can you also add it to iOS and readme?

File: android/src/main/java/com/reactnative/googlecast/GoogleCastModule.java (hunk 1 above)
Comment (danielgssp) [reply android/src/main/java/com/reactnative/googlecast/GoogleCastModule.java]: I'm going to work in IO's circumstances..

**Comment by danielgssp** [comment]
I'm going to work in IO's circumstances..

**Pull request closed without merging**